src/lib/AirspeedValidator/AirspeedValidator.hpp

```diff
@@ -0,0 +1,179 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2019 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * @file AirspeedValidator.hpp
+ * Calculates airspeed from differential pressure and checks if this airspeed is valid.
+ */
+
+#pragma once
+
+#include <airspeed/airspeed.h>
+#include <ecl/airdata/WindEstimator.hpp>
+#include <uORB/topics/wind_estimate.h>
+
+
+using matrix::Dcmf;
+using matrix::Quatf;
+using matrix::Vector2f;
+using matrix::Vector3f;
+
+using namespace time_literals;
+
+struct airspeed_validator_update_data {
+	uint64_t timestamp;
+	float airspeed_indicated_raw;
+	float airspeed_true_raw;
+	uint64_t airspeed_timestamp;
+	float lpos_vx;
+	float lpos_vy;
+	float lpos_vz;
+	bool lpos_valid;
+	float lpos_evh;
+	float lpos_evv;
+	float att_q[4];
+	float air_pressure_pa;
+	float air_temperature_celsius;
+	float accel_z;
+	float vel_test_ratio;
+	float mag_test_ratio;
+	bool in_fixed_wing_flight;
+};
+
+class AirspeedValidator
+{
+public:
+	AirspeedValidator();
```

Comment:
```suggestion
	AirspeedValidator() = default;
```
And you can remove the empty definition.